src/test/java/org/zalando/pazuzu/ResolvedFeatureApiTest.java

```diff
@@ -0,0 +1,71 @@
+package org.zalando.pazuzu;
+
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ResolvedFeatureApiTest extends AbstractComponentTest {
```

Comment:
missing tests :
- empty request
- feature not found
- recursive dependencies (21->22->23)